packages/salesforcedx-vscode-core/src/commands/isvdebugging/bootstrapCmd.ts

```diff
@@ -109,9 +109,9 @@ export class IsvDebugBootstrapExecutor extends SfdxCommandletExecutor<{}> {
           'isv_debug_bootstrap_step2_configure_project_retrieve_namespace'
         )
       )
-      .withArg('force:data:soql:query')
+      .withArg('data query')
       .withFlag('--query', 'SELECT NamespacePrefix FROM Organization LIMIT 1')
-      .withFlag('--targetusername', data.sessionId)
+      .withFlag('--target-org', data.sessionId)
       .withJson()
       .withLogName('isv_debug_bootstrap_configure_project_retrieve_namespace')
```

Comment:
It seems like we should also change/get rid of this flag...
"The loglevel flag is no longer in use on this command. You may use it without error, but it will be ignored.\nSet the log level using the SFDX_LOG_LEVEL environment variable."

Reply:
Yes! I'm not clear why are we using the --loglevel flag in these commands So not touching them for now. And having the flag --loglevel does not throwing any warnings so we should just keep it for now.